### DISCUSSION TITLE
Restlet Framework update

### DISCUSSION THREAD

**Discussion by jlouvel** [restlet/restlet-framework-java#1405] [General]
Dear Community,

Over the past few years, we unfortunately have not been able to renew our R&D efforts to move the open source Restlet Framework forward. We welcome community contributions to handle [pending pull requests](https://github.com/restlet/restlet-framework-java/pulls) or work on releasing a maintenance release with updated library dependencies for example.

While the [GitHub repository](https://github.com/restlet/restlet-framework-java/) for this project will stay as it is for now, we do need to close [the mailing list based on Google Groups](mailto:framework-discuss@restlet.org) early in 2024. Any future discussion should be directed to [this GitHub Discussions page](https://github.com/restlet/restlet-framework-java/discussions).

In addition, if you still have projects pointing to https://maven.restlet.com, please update them to point to https://maven.restlet.talend.com as we will be discontinuing the restlet.com domain early in 2024 as well.

Best regards,
Jerome Louvel

**Comment by kerbymart** [comment]
Hi, @jlouvel how about the restlet.org domain?

**Reply by jlouvel** [reply]
Hi @kerbymart This is the same for restlet.org domain and especially web site/server serving under it.

**Comment by kerbymart** [comment]
Hi @jlouvel! It'd be awesome to have a community-powered Restlet.org page. A space where our community can come together to build the website, share guides, documentation, and blog posts. A place that stands on its own, independent of enterprise sponsors, truly centered around the community. What do you think?

**Reply by jlouvel** [reply]
Hi @kerbymart Thanks for the various message and our apologizes for the slow replies. @thboileau and I are preparing an update message to the community including our plans for the maintenance release of 2.4 and next versions 2.5+ Please stay tuned, and we will definitely get the refreshing the Restlet website. One area we could use for community driven content is the GitHub wiki, but also the the website itself that is hosted on GitHub pages here: https://github.com/restlet/restlet.github.io
